docs/pages/access-controls/guides/device-trust.mdx

```diff
@@ -206,17 +208,46 @@ Device trust has the following modes of operation, represented by the
   Additionally, it requires a trusted device for all SSH, Database and
   Kubernetes connections.
 
-Enterprise clusters run in `optional` mode by default. Changing the mode to
-`required` will enforce a trusted device for all SSH, Database and Kubernetes
-accesses.
-
 <Admonition type="warning" title="Web UI">
-The Web UI is not capable of trusted device access during the device trust
-preview. Only `tsh` and Teleport Connect are able to fulfill device mode
-`required`.
+  The Web UI is not capable of trusted device access during the device trust
+  preview. Only `tsh` and Teleport Connect are able to fulfill device mode
+  `required`.
+</Admonition>
+
+### Per-Role
+
+To enforce authenticated device checks for a specific role, update the role as the following:
+
+```diff
+kind: role
+version: v6
+metadata:
+  name: example-role
+spec:
+  options:
+    # require authenticated device check for this role
++   device_trust_mode: "required" # add this line
```

Comment:
Do we have a reference page for role options? Let's update it if we do. (Maybe @alexfornuto knows?)

Reply:
Yes, we do. Please update `role-spec.mdx` and `access-controls/reference.mdx`.

Reply:
Resolved in the following commit.